### PR TITLE
Nav Title: casing consistency

### PR DESCRIPTION
**Pull request by bluk16**

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift [hunk 1]
```diff
@@ -135,24 +135,24 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
         case .account:
             self.pushAccountSelection()
         case .orderAction:
-            self.selectionViewController.title = "Select " + ticketRow.getTitle(forOrder: self.order)
+            self.selectionViewController.title = "Select " + ticketRow.getTitle(forOrder: self.order).lowercased()
```
Comment (jsom) [inline TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift]: Maybe move this in to the `pushOrderCapabilitiesSelection` function so you don't have to repeat this code

**Comment by bluk16** [comment]
For marginType screen (FidelityV1) and leverage screen (FX), they don't use the orderCapabilitiesSelection()

**Comment by jsom** [comment]
@bluk16 did you test the Yahoo screen as well? Looks good?

**Comment by bluk16** [comment]
Looks good now

**Pull request merged**